scripts/addresses.ts

```diff
@@ -29,3 +29,13 @@ export const HOPR_MINTER: { [key in Networks]: string } = {
   kovan: undefined,
   private: '0x0a67180CF519aDF27f1FD32F7255bBa00B536FC6',
 }
+
+export const HOPR_FAUCET: { [key in Networks]: string } = {
+  mainnet: undefined,
+  morden: undefined,
+  ropsten: undefined,
+  rinkeby: undefined,
+  goerli: undefined,
+  kovan: '0xa75159bC8936d471d8E272a358d840a899447dCa',
+  private: '0x6c97048D67c39ADCe38bbB228fc1bA415fc8f096',
```

Comment:
In other iteration we should produce this from the `migration` script. For now it's fine, but for future reference.

Reply:
Created issue https://github.com/hoprnet/hopr-ethereum/issues/68 to keep this tracked.